### PR TITLE
Organize the pages concerning the meetings

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
I've moved the meeting minutes down a layer to make the `meeting`
directory less messy.  I've added a link to the meeting page
from the community page and have added an agenda for the
next meeting.

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@rhatdan PTAL
@baude @afbjorklund please verify that you're still OK with the topics you volunteered for and that I gave you the amount of time that you'd like to lead that discussion.

**Comment by afbjorklund** [comment]
@TomSweeneyRedHat 
> please verify that you're still OK with the topics you volunteered for and that I gave you the amount of time that you'd like to lead that discussion.

~~Somehow I couldn't find the agenda / planning for _next_ meeting (Nov 3), only the previous one ?~~ But I was planning to do a boot2podman presentation, thinking about 10 min presentation and 5 minute demo.

* docker-machine demo: https://asciinema.org/a/366096

* podman-machine demo: https://asciinema.org/a/366097

Won't expand so much on the **details** of bootstrapping a linux distribution for podman or **how** the varlink connection actually worked... Mostly presenting the background and some "boot2podman" project details.

The rest is on the web site, so will just add links to it for the interested...

* https://github.com/boot2podman/boot2podman/blob/master/building_podman.md

* https://github.com/boot2podman/boot2podman/blob/master/building_varlink.md

Basically a post-mortem, will just redirect people to "set up their own VM"

----

EDIT: Of course, I just found the agenda _after_ I commented. Looks like my plan was correct (15 min)

> 11:05 -> 11:20 - Anders Björklund - boot2podman/podman-machine (Basically a varlink post-mortem)

OOPS: Recorded the demo using the old and broken "direct ssh" method. Was wonder why it failed later.

Will improve upon it, either by making varlink the default in the tool or remembering the --varlink parameter.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan can you merge this if it LGTY please?

**Review by rhatdan — APPROVED** [review]
